lib/field-locale.ts

```diff
@@ -23,8 +26,12 @@ export default class FieldLocale implements FieldAPI {
     this.type = info.type
     this.required = info.required
     this.validations = info.validations
-    this.items = info.items
-
+    if (info.type === 'Array') {
```

Comment:
Does the condition change anything? If the type is not array, `items` is anyway initialized with `undefined`. So, this only prevents issues if we send an invalid connect message, right?

Reply:
Typescript doesn't like if we do it in a straightforward way:
```js
this.items = info.items
this.linkType = info.linkType
```
> Property 'items' does not exist on type 'BasicFieldInfo'.ts(2339)

We can do it like this:
```js
this.items = (info as ArrayFieldAPI).items
```
But I wouldn't export `ArrayFieldAPI` just for this one case.

So yeah, just an extra check.

Reply:
Btw, we can delete `items` and `linkType` if `type` is not an Array or Link 🤔 Wdyt?